app-frontend/src/app/components/scenes/sceneItem/sceneItem.html

```diff
@@ -27,10 +27,19 @@
   </div>
   <div class="list-group-right">
     <rf-toggle-old
-      ng-if="$ctrl.onSelect && !$ctrl.isDisabled"
+      ng-if="$ctrl.onSelect && !$ctrl.isDisabled && $ctrl.hasDownloadPermission()"
       model="$ctrl.selectedStatus"
       on-change="$ctrl.toggleSelected($event)">
     </rf-toggle-old>
+    <a class="scene-item-lock" ng-if="!$ctrl.hasDownloadPermission()">
+      <span class="icon-locked"
+        tooltips
```

Comment:
Stick the tooltip on the containing `<a>`, right now it requires hovering over the much smaller icon which is easy to miss.

Reply:
also, I don't think it should be an <a> since it isn't actually a hyperlink
<div> should work fine. remove element type from the style